lte/gateway/c/core/oai/tasks/sctp/sctp_itti_messaging.c

```diff
@@ -55,6 +55,9 @@ status_code_e sctp_itti_send_new_association(
   MessageDef* msg =
       DEPRECATEDitti_alloc_new_message_fatal(TASK_SCTP, SCTP_NEW_ASSOCIATION);
 
+  sctp_new_peer_t* new_sctp_peer = &(msg->ittiMsg.sctp_new_peer);
+  memset((void*) new_sctp_peer, 0, sizeof(sctp_new_peer_t));
+
```

Comment:
what is the purpose of this addition?

Reply:
it was added on  PR #6972, which might be added initialization. 

it was not needed here, DEPRECATEDitti_alloc_new_message_fatal will initialize to 0.

resolved and removed memset.